src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs

```diff
@@ -53,7 +53,10 @@ public static StandardGlyphGroup GetStandardGlyphGroup(this Glyph glyph)
                 case Glyph.EnumInternal:
                     return StandardGlyphGroup.GlyphGroupEnum;
 
-                case Glyph.EnumMember:
+                case Glyph.EnumMemberPublic:
+                case Glyph.EnumMemberProtected:
+                case Glyph.EnumMemberPrivate:
+                case Glyph.EnumMemberInternal:
```

Comment:
Even though all enum members are public, this inconsistency with all the rest of our glyphs just made later work more annoying.